Detect oldValue, newValue and Operation type in Glide List

SrijanPatwa opened the pull request:
# PR Description: 


# Pull Request Checklist

## Overview
- [x] I have read and understood the [CONTRIBUTING.md](CONTRIBUTING.md) guidelines
- [x] My pull request has a descriptive title that accurately reflects the changes
- [x] I've included only files relevant to the changes described in the PR title and description
- [x] I've created a new branch in my forked repository for this contribution

## Code Quality
- [x] My code is relevant to ServiceNow developers
- [x] My code snippets expand meaningfully on official ServiceNow documentation (if applicable)
- [x] I've disclosed use of ES2021 features (if applicable)
- [x] I've tested my code snippets in a ServiceNow environment (where possible)

## Repository Structure Compliance
- [x] I've placed my code snippet(s) in one of the required top-level categories:
  - `Core ServiceNow APIs/`
  - `Server-Side Components/`
  - `Client-Side Components/`
  - `Modern Development/`
  - `Integration/`
  - `Specialized Areas/`
- [x] I've used appropriate sub-categories within the top-level categories
- [x] Each code snippet has its own folder with a descriptive name

## Documentation
- [x] I've included a README.md file for each code snippet
- [x] The README.md includes:
  - Description of the code snippet functionality
  - Usage instructions or examples
  - Any prerequisites or dependencies
  - (Optional) Screenshots or diagrams if helpful

## Restrictions
- [x] My PR does not include XML exports of ServiceNow records
- [x] My PR does not contain sensitive information (passwords, API keys, tokens)
- [x] My PR does not include changes that fall outside the described scope

ravichandra1998g reviewed (changes requested):
Thank you for submitting the idea.

I know Glidelist store comma separated sys ids, but would it be possible for you to show the display values instead of sys ids?

SrijanPatwa commented:
> Thank you for submitting the idea.
> 
> I know Glidelist store comma separated sys ids, but would it be possible for you to show the display values instead of sys ids?

Hi @ravichandra1998g , Updated the script and added screenshots. Thanks

ravichandra1998g reviewed (changes requested):
Hello @SrijanPatwa,
Thank you for making the changes. looks good now!

Can you please change the script include name from "scriptUtil" to more meaningful name?

SrijanPatwa commented:
> Hello @SrijanPatwa, Thank you for making the changes. looks good now!
> 
> Can you please change the script include name from "scriptUtil" to more meaningful name?

Hi @ravichandra1998g , Updated. Please review again, thanks
